kibana/Dockerfile

```diff
@@ -1,55 +1,50 @@
-FROM rhscl/nodejs-6-rhel7
+FROM rhscl/nodejs-10-rhel7 AS builder
+
+ENV KIBANA_VERSION=6.8.0 \
+    KIBANA_DOWNLOAD_URI=https://artifacts.elastic.co/downloads/kibana/
```

Comment:
internal builds won't be able to pull from `https://artifacts.elastic.co`

Reply:
what should we do here then, download and add it to the repo?

Reply:
For fluentd and rsyslog, we are downloading the source code, adding the expanded source code to git, and building the container from that expanded source code.  We are also planning to do the same with with Elasticsearch.  I suppose we could add the kibana tarball .tar.gz file to git, but that is generally frowned upon, and is probably against openshift practice - @bparees

Reply:
Wouldn't we also expand the source in the same manor as the other components if we are going to go the vendoring approach?

Reply:
adding binaries to our git repos is definitely against our practices.